cognite_toolkit/_cdf_tk/apps/_profile_app.py

```diff
@@ -34,3 +35,33 @@ def asset_centric(
                 verbose,
             )
         )
+
+    @staticmethod
+    def transformations(
+        ctx: typer.Context,
```

Comment:
The `ctx` argument seems to not be used.
Remove? 
Or will it be used in later iterations?

Reply:
It gives ctx from typer, but it is not needed at this stage.